pkg/reconciler/image/build_needed.go

```diff
@@ -61,7 +61,7 @@ func builtWithBuildpacks(build *v1alpha1.Build, buildpacks v1alpha1.BuildpackMet
 	return true
 }
 
-func builtWithStack(build *v1alpha1.Build, runImage string) bool {
+func builtWithClusterStack(build *v1alpha1.Build, runImage string) bool {
```

Comment:
I think this might be the result of a aggressive goland rename and might be a bit confusing if we merge it.